app/models/dance.rb

```diff
@@ -14,7 +14,9 @@ class Dance < ActiveRecord::Base
     elsif user.admin?
       all
     else
-      where('publish= true OR user_id= ?', user.id)
```

Comment:
Huh. Okay, I think the rails5 branch is ready to merge, which will collide with this, but hopefully be the clean, long term solution to this.